DataDog/rosie
# [STAT-157] update severities

## What problem are you trying to solve?

- We have a new severity value, `NOTICE`, that isn’t supported in Rosie yet and some old ones that can be deprecated, `INFORMATIONAL` and `CRITICAL`
- We aren’t using the Severity type on server requests similar to other fields that were done in #41

## What is your solution?

- Add `NOTICE` severity value
- Update the usage of severity from type `String` to `Severity`
  - this includes changes to test files
- Update `INFORMATIONAL` use-cases to use `NOTICE`
- Update `CRITICAL` use-cases to use `ERROR`
- Remove `INFORMATIONAL` and `CRITICAL` values

## Alternatives considered

Considered deprecated the two enums that I removed, but the impact to remove them was minimal imo.

## What the reviewer should know

This was a task that could've been completed in #41, but to unblock the frontend and to release the server, I decided to commit this separately.
